SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -67,7 +67,41 @@ public FormSubmissionsUseCase(IDatabaseGateway databaseGateway, IMongoGateway mo
             return foundSubmission?.ToDomain().ToResponse();
         }
 
-        public static FilterDefinition<CaseSubmission> GenerateFilter(QueryCaseSubmissionsRequest request)
+        public void ExecuteDelete(string submissionId, DeleteCaseSubmissionRequest request)
+        {
+            _ = GetSanitisedWorker(request.DeletedBy);
+
+            var submissionToBeDeleted = _mongoGateway.LoadRecordById<CaseSubmission>(_collectionName, ObjectId.Parse(submissionId));
+
+            if (submissionToBeDeleted == null)
+            {
+                throw new DeleteSubmissionException($"Submission with ID {submissionId} not found");
+            }
+
+            if (submissionToBeDeleted.Deleted == true)
+            {
+                throw new SubmissionAlreadyDeletedException($"Submission with ID { submissionId} has already been deleted");
+            }
+
+            if (submissionToBeDeleted.FormId?.ToLower() != FormIdName.ChildCaseNote && submissionToBeDeleted.FormId?.ToLower() != FormIdName.AdultCaseNote)
```

Comment:
Should this be an or?

Won't this always be false?

Reply:
It kind of feels a bit backwards, but using `OR` would mean that when formId is `child-case-note` for example, then 
submissionToBeDeleted.FormId?.ToLower() != `child-case-note` is `false`
submissionToBeDeleted.FormId?.ToLower() != `adult-case-note` is `true`
=> `true`, throws exception

anything other than accepted values will result in `true` && `true` => exception